core/sqf/sql/scripts/get_libhdfs_files

```diff
@@ -0,0 +1,186 @@
+if [[ -z ${LIBHDFS_TEMP_DIR} ]]; then
+  LIBHDFS_TEMP_DIR=${MY_SQROOT}/sql/libhdfs_files
+fi
+LOGFILE=${LIBHDFS_TEMP_DIR}/build.log
+
+# Hadoop source tar file to build libhdfs from
+HADOOP_SRC_MIRROR_URL=https://archive.apache.org/dist/hadoop/common/hadoop-2.6.0
+HADOOP_ID=hadoop-2.6.0
+HADOOP_SRC_ID=${HADOOP_ID}-src
+HADOOP_SRC_TAR=${HADOOP_SRC_ID}.tar.gz
+
+# files to build required version of Google Protocol Buffers
+PROTOBUF_MIRROR_URL=https://github.com/google/protobuf/releases/download/v2.5.0
+PROTOBUF_ID=protobuf-2.5.0
+PROTOBUF_TAR=${PROTOBUF_ID}.tar.gz
+
+# result of protobuf build
+PROTOBUF_TGT_ID=protobuf-tgt
+
+# Directories to copy the built libhdfs library and corresponding include file
+TGT_INC_DIR=$MY_SQROOT/export/include
+TGT_LIB_DIR=$MY_SQROOT/export/lib${SQ_MBTYPE}
+
+FORCE_BUILD=false
+VERBOSE=false
+
+USAGE="Usage $0 [ -f | --force ]\
+                [ -v | --verbose ]\
+                [ -d <temp dir> | --tempDir <temp dir> ]"
+
+while [[ $# > 0 ]]
+do
+  arg="$1"
+
+  case $arg in
+    -f|--force)
+       FORCE_BUILD=true
+       ;;
+
+    -v|--verbose)
+       VERBOSE=true
+       ;;
+
+    -d|--tempDir)
+       shift
+       if [[ $# < 1 ]]; then
+         echo "Expecting argument after -d or --tempDir"
+         exit 1
+       fi
+       LIBHDFS_TEMP_DIR="$1"
+       ;;
+
+    *)
+       echo "Unknown command line option: $arg"
+       echo "Usage $0 [ -f | --force ]"
+       exit 1
+    ;;
+  esac
+
+  shift
+done
+
+
+if [[ $FORCE_BUILD == true || \
+      ! -e ${TGT_INC_DIR}/hdfs.h || \
+      ! -e ${TGT_LIB_DIR}/libhdfs.so ]]; then
+
+  if [ ! -d $LIBHDFS_TEMP_DIR ]; then
+    mkdir $LIBHDFS_TEMP_DIR
+  fi
+
+  cd $LIBHDFS_TEMP_DIR
+
+  if [[ ! -f ${PROTOBUF_TAR} ]]; then
+    echo "Downloading Google Protocol Buffers..." | tee -a ${LOGFILE}
+    wget ${PROTOBUF_MIRROR_URL}/${PROTOBUF_TAR} >${LOGFILE}
+  fi
+
+  if [[ $FORCE_BUILD == true ]]; then
+    rm -rf ${LIBHDFS_TEMP_DIR}/${PROTOBUF_ID}
+    rm -rf ${LIBHDFS_TEMP_DIR}/${PROTOBUF_TGT_ID}
+  fi
+
+  if [[ ! -d ${PROTOBUF_ID} ]]; then
+    echo "Unpacking Google Protocol Buffer tar file..." | tee -a ${LOGFILE}
+    rm -rf ${LIBHDFS_TEMP_DIR}/${PROTOBUF_TGT_ID}
+    tar -xzf ${PROTOBUF_TAR} >>${LOGFILE}
+  fi
+
+  if [[ ! -d $PROTOBUF_TGT_ID ]]; then
+    cd ${PROTOBUF_ID}
+    echo "Building Google Protocol Buffers, this could take a while..." | tee -a ${LOGFILE}
+    if [[ $VERBOSE == true ]]; then
+      ./configure --prefix=${LIBHDFS_TEMP_DIR}/${PROTOBUF_TGT_ID} 2>&1 | tee -a ${LOGFILE}
+    else
+      ./configure --prefix=${LIBHDFS_TEMP_DIR}/${PROTOBUF_TGT_ID} 2>&1 >>${LOGFILE}
+    fi
+    if [[ $? != 0 ]]; then
+      echo "Error during configure step, exiting" | tee -a ${LOGFILE}
+      exit 1
+    fi
+    make 2>&1 >>${LOGFILE}
+    if [[ $? != 0 ]]; then
+      echo "Error during make step, exiting" | tee -a ${LOGFILE}
+      exit 1
+    fi
+    # skip the tests
+    # make check 2>&1 >>${LOGFILE}
+    if [[ $? != 0 ]]; then
```

Comment:
Probably should comment out this "if" block too since the "make check" is commented out. Though it is harmless (the "if" should always be false).